src/services/ApiService.js

```diff
@@ -23,4 +23,8 @@ export class ApiService {
   createProject(project) {
     return HttpClient.post('/projects', project, this.headers);
   }
+
+  createProjectMember(projectMember) {
+    return HttpClient.post('projectmember', projectMember, this.headers);
```

Comment:
Add a forward slash here otherwise the request will throw an error. 
```suggestion
    return HttpClient.post('/projectmember', projectMember, this.headers);
```